CHANGES.rst

```diff
@@ -1,198 +1,194 @@
-Changelog
-=========
-
-
```

Comment:
If you want to add the changelog contents from #715, then #715 can just be closed as the other changes in it are made mute by this PR.

Reply:
cherry picked your changelog commit